tests/testthat/test-compiler-ghpages.R

```diff
@@ -45,3 +45,18 @@ test_that("animint2pages raises an error if no GitHub token is present", {
   do.call(Sys.setenv, as.list(env.old))
   file.copy(config.old, config.file, overwrite = TRUE)
 })
+
+test_that("animint2pages() default branch is gh-pages", {
+  whoami <- suppressMessages(gh::gh_whoami())
+  if (is.null(whoami)) {
+    owner <- "tdhock"
+  } else {
+    owner <- whoami[["login"]]
+  }
+  local_repo_path <- tempfile(pattern = "repo_clone_")
+  test_viz_repo <- animint2pages(viz, github_repo = "a2p_ghp_test")
+  gert::git_clone(url = paste0("https://github.com/", owner, "/a2p_ghp_test.git"), path = local_repo_path)
+  # Check the default branch after clone
+  default_branch <- gert::git_branch(repo = local_repo_path)
+  expect_equal(default_branch, "gh-pages")
```

Comment:
can this test be added inside the "animint2pages() returns owner/repo string" block above?